src/System.Buffers.Primitives/System/Internal/ManagedBufferPool.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace System.Buffers.Internal
 {
-    internal sealed class ManagedBufferPool : BufferPool
+    internal sealed partial class ManagedBufferPool : BufferPool
```

Comment:
Would the same apply to this:
https://github.com/dotnet/corefxlab/pull/1463/files#diff-82f70dec2e02480d89f3a6769347bdd9R8